### DISCUSSION TITLE
Initial Proposal: Reducing Cronos Mainnet Minimum Base Fee

### DISCUSSION THREAD

**Discussion by lezzokafka** [crypto-org-chain/cronos#1338] [Ideas]
# Abstract

In response to community feedback and the ongoing evolution of the blockchain network, one of the key parameters the Cronos team has been monitoring is the base transaction fee, which is a primary factor in the cost of executing transactions on the Cronos Chain. With this in mind, we propose an update to the "fee market" parameters:

- Reducing the min_gas_price from `10,000` gwei to: `5,000` gwei

In short, this will establish a lower floor value for the transaction "base fee", effectively reducing the transaction fee by up to 50%. 

# Technical details

Cronos employs a fee market module that incorporates a [dynamic fee structure](https://medium.com/cronos-chain/network-upgrade-on-cronos-testnet-v0-7-0-84e962f10e47). The transaction base fee for the next block is dynamically calculated at each block, depending on overall network utilization. 

Specifically, this fee market module and formula was designed to calculate the base fee at each block is similar to [EIP-1559](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1559.md). As a result, transaction fees will rise when the network is busy and reduced when there is excess capacity. 
Proposed changes
On a technical side, we suggest updating the following network parameter:

- Reducing the min_gas_price from `10,000` gwei to: `5,000` gwei

```
    "changes": [
        {
            "subspace": "feemarket",
            "key": "MinGasPrice",
            "value": "5000000000000.000000000000000000"
        }
  ```

In particular, this change will effectively reduce the lower bound of the base transaction fee by up to 50%, which allows a lower floor value of the base fee to be reached and effectively reducing the transaction fee by up to 50% when the network is not busy. This proposed change is intended to foster network activity with a reasonable and competitive base transaction fee while ensuring automatic adjustment of transaction fees in response to traffic variations.  

# Testing

The proposed change has been tested in internal testnets and will be tested on the Cronos testnet to ensure that the expected outcomes are achieved.

**Comment by Markmanus** [comment]
Amazing proposal. Thank you for listening to the community.

**Comment by lezzokafka** [comment]
Hello #CROFam - The proposal is now active under proposal id `8` 
For details : https://rest.cronos.org/cosmos/gov/v1beta1/proposals/8

**Reply by Azizcorolla1z** [reply]
That sound good

**Comment by pmbrsantos** [comment]
Sounds like a great ideia to get base fee back to where it was some weeks ago.
Don't think no one will say no, unless the Validators count on that revenue.

**Comment by Azizcorolla1z** [comment]
Amazing proposal. Thank you for listening to the community.

**Comment by Raycarmonannnn** [comment]
if we need to do this and also do mass burning to see cro where he once was

**Reply by Azizcorolla1z** [reply]
and It will be great

**Comment by Azizcorolla1z** [comment]
Amazing proposal. Thank you for listening to the community.

**Reply by Azizcorolla1z** [reply]
Thank you
